src/qautostart.cpp

```diff
@@ -31,102 +31,86 @@
 #include <QFile>
 #include <QDir>
 
```

Comment:
Constructors in C+++ are evil. But let it be. It doesn't hurt. Is it?

Reply:
It's doesn't do anything. I don't think it should be added just because 'it doesn't hurt'. If you need it, just add it back again!

Reply:
OK. Remove it. You convinced me.